docs/source/quickstart.md

```diff
@@ -135,4 +135,51 @@ python -u "$@" --node_rank $SLURM_NODEID -u ${PATH_TO_MAMMOTH}/train.py \
     -tensorboard -tensorboard_log_dir ${LOG_DIR}/${EXP_ID}
 ```
 
-A complete example of training on the Europarl dataset is available at [MAMMOTH101](examples/train_mammoth_101.md).
\ No newline at end of file
+Explanation of Command:
+   - `python -u "$@"`: Initiates the training script using Python.
+   - `--node_rank $SLURM_NODEID`: Specifies the node rank using the environment variable provided by Slurm.
+   - `-u ${PATH_TO_MAMMOTH}/train.py`: Specifies the path to the Mammoth training script.
+   - `-config ${CONFIG_DIR}/your_config.yml`: Specifies the path to your configuration file.
+   - `-save_model ${SAVE_DIR}/models/${EXP_ID}`: Defines the directory to save the trained models, incorporating an experiment identifier (`${EXP_ID}`).
+   - `-master_port 9974 -master_ip $SLURMD_NODENAME`: Sets the master port and IP for communication.
+   - `-tensorboard -tensorboard_log_dir ${LOG_DIR}/${EXP_ID}`: Enables TensorBoard logging, specifying the directory for TensorBoard logs.
+
+Your training process has been initiated through the Slurm manager, leveraging the specified configuration settings. Monitor the progress through the provided logging and visualization tools. Adjust parameters as needed for your specific training requirements. You can also run the command on other workstations by modifying the parameters accordingly.
+
+
+
+### Step 4: Translate
+
+Now that you have successfully trained your multilingual machine translation model using Mammoth, it's time to put it to use for translation. 
+
+```bash
+python3 -u $MAMMOTH/translate.py \
+  --config "${CONFIG_DIR}/your_config.yml" \
+  --model "$model_checkpoint" \
+  --task_id  "train_$src_lang-$tgt_lang" \
+  --src "$path_to_src_language/$lang_pair.$src_lang.sp" \
+  --output "$out_path/$src_lang-$tgt_lang.hyp.sp" \
+  --gpu 0 --shard_size 0 \
+  --batch_size 512
+```
+
+Follow these configs to translate text with your trained model.
+
+- Provide necessary details using the following options:
+   - Configuration File: `--config "${CONFIG_DIR}/your_config.yml"`
+   - Model Checkpoint: `--model "$model_checkpoint"`
+   - Translation Task: `--task_id "train_$src_lang-$tgt_lang"`
+
+- Point to the source language file for translation:
+   `--src "$path_to_src_language/$lang_pair.$src_lang.sp"`
+- Define the path for saving the translated output: `--output "$out_path/$src_lang-$tgt_lang.hyp.sp"`
+- Adjust GPU and batch size settings based on your requirements: `--gpu 0 --shard_size 0 --batch_size 512`
```

Comment:
shard_size needs to be removed altogether, this is a bug (cf. #7). Keep as is for now, it should be more or less fixed in #49